src/modules/transaction/utils/transaction.js

```diff
@@ -111,6 +114,9 @@ const getElementsTxAsset = (desktopAsset, moduleAssetId) => {
     case registerDelegate: {
       return {
         username: desktopAsset.username,
```

Comment:
the form  changed to name please also change from here
```suggestion
        name: desktopAsset.name,
```

Reply:
This is not something I choose, Lisk SDK gets to choose this parameter name.